test/config/presets.spec.js

```diff
@@ -147,7 +147,7 @@ describe('config/presets', () => {
       const res = await presets.resolveConfigPresets(config);
       expect(res).toMatchSnapshot();
       const rule = res.packageRules[0];
-      expect(rule.automerge).toEqual(true);
+      expect(rule.automerge).toBe(true);
```

Comment:
What's the difference? Looks like there's practically no difference?

Reply:
There's no difference, changed it to maintain consistency